### PR TITLE
Financial instrument rw neo4j

### PR DESCRIPTION
**Pull request by danielcopaciu**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -50,7 +51,7 @@ Response format for current version:
 ### PUT
 The only mandatory field is the uuid, and the alternativeIdentifier uuids (because the uuid is also listed in the alternativeIdentifier uuids list). The uuid in the body must match the one used on the path.
 
-Every request results in an attempt to update that financial instrument: unlike with GraphDB there is no check on whether the person already exists and whether there are any changes between what's there and what's being written. We just do a MERGE which is Neo4j for create if not there, update if it is there.
+Every request results in an attempt to update that financial instrument: unlike with GraphDB there is no check on whether the financial instrument already exists and whether there are any changes between what's there and what's being written. We just do a MERGE which is Neo4j for create if not there, update if it is there.
```
Comment (nwrigh) [inline README.md]: Can we remove references to GraphDB as this functionality wasn't in GraphDB

File: financialinstruments/query.go [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+				MERGE (t)<-[:IDENTIFIES]-(i)
+				set i : %s`
+
+const writeStatement = `MERGE (n:Thing{uuid: {uuid}})
+			set n={props}
+			set n :Concept
+			set n :FinancialInstrument
+			set n :Equity`
+
+const deleteEntityRelationshipsStatement = `MATCH (t:Thing {uuid:{uuid}})
```
Comment (Nesze) [inline financialinstruments/query.go]: It depends on the final model, but should we also delete the existing organization relationship?

File: financialinstruments/query.go [hunk 3]
```diff
@@ -0,0 +1,55 @@
...
+					MERGE (fi)-[:ISSUED_BY]->(o)`
+
+const clearIdentifierStatement = `MATCH (p:Thing {uuid: {uuid}})
+				OPTIONAL MATCH (p)<-[ir:IDENTIFIES]-(i:Identifier)
+				REMOVE p:Concept
+				REMOVE p:FinancialInstrument
+				REMOVE p:Equity
+				DELETE ir, i
+				SET p={props}`
+
```
Comment (Nesze) [inline financialinstruments/query.go]: Should we tidy up the relation to the affiliated organization, too?

**Comment by Nesze** [comment]
lgtm

**Pull request merged**